packages/replay-next/src/suspense/LogPointAnalysisCache.ts

```diff
@@ -19,6 +19,7 @@ import { mappedExpressionCache } from "./MappedExpressionCache";
 
 export type LogPointAnalysisResult = {
   executionPoint: ExecutionPoint;
+  failed?: boolean;
```

Comment:
nit: I don't like optional attributes. We could mark this one non-optional if we assigned like `failed: result.failed === true`